src/github-issues-prs.ts

```diff
@@ -93,6 +95,21 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 		}
 	}
 
+	private async createIssue() {
+		let remotes: GitRemote[];
+
+		try {
+			remotes = await this.getGitHubRemotes();
+		} catch (err) {
+			return false;
+		}
+
+		// take first one
+		await open(getGitHubUrl(remotes[0].url) + '/issues/new');
```

Comment:
The first one might not be the right one. Maybe you could let the user choose one with window.showQuickPick() when there are multiple remotes?